### ISSUE TITLE
Weasyprint: language issue

### ISSUE THREAD

**Issue by Marcus-D** [cuckoosandbox/cuckoo#1868]
I am working with weasyprint 0.40 on Cuckoo Sandbox 2.0.4 on Ubuntu 16.04. I am getting this:
 ERROR: Failed to load font at "data:application/font-woff2;charset=utf-8;base64,  

Do I need change the font? I already tried reinstalling the weasyprint. I tried upgrading weasyprint. I am still getting the same error.

**Comment by Marcus-D** [comment]
Seems like weasyprint 0.40 is not compatible with Cuckoo 2.0.4. I am not sure how that is. My workaround is reverting back to weasyprint 0.39.

**Comment by jbremer** [comment]
Yes, the version we tested was `weasyprint==0.36`, see also our `setup.py` file. We'll have to re-evaluate the latest versions and make sure it works with Cuckoo etc. Thanks!
